core/templates/pages/exploration-player-page/exploration-player-page.mainpage.html

```diff
@@ -14,13 +14,6 @@
         }
       }
     </style>
-    <!-- Prefetch files to enable the MathExpressionInput interaction to work
-         even in offline mode. -->
-    <link rel="prefetch" href="/third_party/static/MathJax-2.7.5/jax/output/HTML-CSS/jax.js?V=2.7.5">
```

Comment:
Isn't something like this not required for the new math interactions?

Reply:
Yes, the interactions do work offline and this is not needed since MathJax is not used in the new interactions.

Reply:
Do we still need MathJax until @donosco98 also does his deprecation of Math RTE components?

Reply:
@seanlip According to [this](https://github.com/oppia/oppia/pull/10331#discussion_r471910063), it seems it's fine to remove it in this pr?

Reply:
Oh I see! Didn't see that, thanks for flagging it.

Yup, sounds like it's OK. @donosco98 can you confirm?

Reply:
Yeah, I can confirm its fine to remove it in this PR(since these just pre-fetch the files for math interaction).
The actual removal of MathJax is done in my PR. [here](https://github.com/oppia/oppia/pull/10331/files#diff-b29ad93457f46f58229dccc8d15d65d9L48)